File: 백준/Bronze/2480. 주사위 세개/주사위 세개.js
let fs = require('fs')
const input = fs.readFileSync("/dev/stdin").toString().trim().split(" ");

let answer = 0

if (input[0] === input[1] && input[1] === input[2]) {
    answer = 10000 + parseInt(input[0]) * 1000
} else if (input[0] === input[1] || input[1] === input[2]) {
    answer = 1000 + parseInt(input[1]) * 100
} else if (input[0] === input[2]) {
    answer = 1000 + parseInt(input[2]) * 100
} else {
    input.sort((a, b) => b - a)
    answer = parseInt(input[0]) * 100
}

console.log(answer)

File: 백준/Bronze/2480. 주사위 세개/주사위 세개.py
import sys
input = sys.stdin.readline

nums = [int(x) for x in input().split()]
dict = {}
for i in nums:
    if i in dict:
        dict[i] += 1
    else:
        dict[i] = 1

dict = sorted(dict.items(), key=lambda x : (-x[1], -x[0]))

if dict[0][1] == 3:
    answer = 10000 + dict[0][0] * 1000
elif dict[0][1] == 2:
    answer = 1000 + dict[0][0] * 100
else:
    answer = dict[0][0] * 100

print(answer)